frontend/src/static/js/utils/urls.ts

```diff
@@ -96,6 +101,15 @@ function getContextualQueryStringParams(
   if (colSpec) {
     searchParams.set('columns', colSpec);
   }
+
```

Comment:
Add a new test case to the existing formatOverviewPageUrl [test](https://github.com/GoogleChrome/webstatus.dev/blob/88d31eccb7c2bb8ff69fc5cbf70d6706101407ec/frontend/src/static/js/utils/test/urls.test.ts#L95)  that:

1. should pass in the new column_options and assert that it is added to the url
2. That it can overridden
3. check that it can be cleared

Check these three cases out for inspiration:

https://github.com/GoogleChrome/webstatus.dev/blob/88d31eccb7c2bb8ff69fc5cbf70d6706101407ec/frontend/src/static/js/utils/test/urls.test.ts#L128-L150